tests/regression_tests/trigger_statepoint_restart/results_true.dat

```diff
@@ -0,0 +1,5 @@
+k-combined:
+2.909598E-01 6.161987E-03
+tally 1:
+3.852896E+01
+2.976246E+02
```

Comment:
That was because I was just using the default _get_results() method from the TestHarness superclass, which reads the FIRST statepoint - see testing_harness.py line 90. IMHO it should read the LAST statepoint if there are more than one, but I'd better not change it in case I break other tests.
My test didn't really use this output - instead it was based on asserts in the TriggerStatepointRestartTestHarness. execute_test() method, but I'll add a few extra hoops for the code to jump through to make it load the results from the last statepoint, to avoid further confusion.